docs/src/languages/php/extensions.md

```diff
@@ -60,110 +61,111 @@ You can disable those by adding them to the `disabled_extensions` list.
+| mssql            | *   | *   | *   |     |     |     |     |     |     |
+| mysql            | *   | *   | *   |     |     |     |     |     |     |
+| mysqli           | *   | *   | *   | *   | *   | *   | *   | *   | *   |
+| mysqlnd          | *   | *   | *   | *   | *   | *   | *   | *   | *   |
+| newrelic         |     |     | *   | *   | *   | *   | *   | *   | *   |
+| oauth            |     |     |     | *   | *   | *   | *   | *   |     |
+| odbc             | *   | *   | *   | *   | *   | *   | *   | *   | *   |
+| opcache          |     | *   | *   | *   | *   | *   | *   | *   | *   |
+| openssl          |     |     |     |     |     |     |     |     |     |
+| pdo              | *   | *   | *   | *   | *   | *   | *   | *   | *   |
+| pdo_dblib        | *   | *   | *   | *   | *   | *   | *   | *   | *   |
+| pdo_firebird     | *   | *   | *   | *   | *   | *   | *   | *   |     |
+| pdo_mysql        | *   | *   | *   | *   | *   | *   | *   | *   | *   |
+| pdo_odbc         | *   | *   | *   | *   | *   | *   | *   | *   | *   |
+| pdo_pgsql        | *   | *   | *   | *   | *   | *   | *   | *   | *   |
+| pdo_sqlite       | *   | *   | *   | *   | *   | *   | *   | *   | *   |
+| pdo_sqlsrv       |     |     |     | *   | *   | *   |     | *   |     |
+| http             |     |     | *   |     |     |     |     | *   |     |
+| pgsql            | *   | *   | *   | *   | *   | *   | *   | *   | *   |
+| phar             |     |     |     | *   | *   | *   | *   | *   | *   |
+| pinba            | *   | *   | *   |     |     |     |     |     |     |
+| posix            |     |     |     | *   | *   | *   | *   | *   | *   |
+| propro           |     |     | *   |     |     |     |     |     |     |
+| pspell           | *   | *   | *   | *   | *   | *   | *   | *   | *   |
+| pthreads         |     |     |     |     | *   |     |     |     |     |
+| raphf            |     |     | *   |     |     |     |     | *   | *   |
+| readline         | *   | *   | *   | *   | *   | *   | *   | *   | *   |
+| recode           | *   | *   | *   | *   | *   | *   | *   | *   |     |
+| redis            | *   | *   | *   | *   | *   | *   | *   | *   | *   |
+| shmop            |     |     |     | *   | *   | *   | *   | *   | *   |
+| simplexml        |     |     |     | *   | *   | *   | *   | *   | *   |
+| snmp             | *   | *   | *   | *   | *   | *   | *   | *   | *   |
+| soap             |     |     |     | *   | *   | *   | *   | *   | *   |
+| sockets          |     |     |     | *   | *   | *   | *   | *   | *   |
+| sodium           |     |     |     |     |     | *   | *   | *   | *   |
+| sourceguardian   |     |     |     | *   | *   |     |     |     |     |
+| spplus           | *   | *   |     |     |     |     |     |     |     |
+| sqlite3          | *   | *   | *   | *   | *   | *   | *   | *   | *   |
+| sqlsrv           |     |     |     | *   | *   | *   |     | *   |     |
```

Comment:
Pourquoi?  Again, the 7.3 gap is weird.  Perhaps the docs are wrong on that one?

Reply:
Ah bien vu i've amended the commit